src/test/converter2/behavior/asConstEnum.ts

```diff
@@ -23,3 +25,31 @@ export const ManualEnumHelper: Readonly<{ a: "a" }> = {
 export const WithoutReadonly = {
     a: "a",
 } as { a: "a" };
+
+/* Enum-like objects with numeric literal values */
+
+export const SomeEnumLikeNumeric = {
```

Comment:
Thanks for adding tests! One note - the converter2 tests, unlike the converter tests, do not take snapshots of everything in the test file. Instead, the converted project is passed to a method to do assertions on the converted result. In this case, in `src/test/behaviorTests.ts`, there is an `asConstEnum` function which is responsible for testing that this file gets converted correctly.

Reply:
Ah, I see - I think I fixed it now. Sorry! :)